controller/users.js

```diff
@@ -0,0 +1,164 @@
+import models from '../models';
+
+const { User } = models;
+/**
+ * A class that handles user methods
+* */
+class UserController {
+  /**
+     * registers a user
+     * @param {object} req - request object
+     * @param {object} res - response object
+     * @param{function} next - next function
+     * @returns {object} response object
+     *
+  */
+  static async resgisterUser(req, res, next) {
+    const {
+      firstName, lastName, email, userName, password, confirmPassword, industry, age, employed,
+    } = req.body;
+    const usersObj = await User.create({
+      firstName,
+      lastName,
+      email,
+      userName,
+      password,
+      confirmPassword,
+      age,
+      industry,
+      employed
+    }).catch(next);
+    return res.status(201).json({
+      status: res.statusCode,
+      message: 'user created sucessfully',
+      user: [{
```

Comment:
Just return the object and get rid of the `[]` wrapped around it.
Just like this 👇 
```js
     {
        id: usersObj.id,
        firstName: usersObj.firstName,
        lastName: usersObj.lastName,
        email: usersObj.email,
        userName: usersObj.userName,
      }
```